src/cli/cli_mac_filter.cpp

```diff
@@ -288,6 +288,25 @@ template <> otError MacFilter::Process<Cmd("addr")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli macfilter rss
+ * @code
+ * macfilter rss
+ * 0f6127e33af6b403 : rss -95 (lqi 1)
+ * Default rss: -50 (lqi 3)
+ * Done
+ * @endcode
+ * @par
+ * Provides the following information:
+ * - Listing of all the extended addresses
+ * where the received signal strength (`rss`) has been set to be different from the default value.
+ * The link quality indicator (`lqi`) is also shown. The `rss` and `lqi`
+ * settings map to each other; if you set one, the value of the other gets set automatically.
```

Comment:
Can you avoid using semi-colons.  You can just make them separate sentences

...map to each other.  If you set one...